test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml

```diff
@@ -58,6 +58,8 @@ spec:
             - mountPath: /var/lib/kubelet/plugins
               mountPropagation: Bidirectional
               name: plugins-dir
+            - mountPath: /csi-data-dir
+              name: csi-data-dir
```

Comment:
Requiring this path to be created by Kubernetes is a breaking change. Do you think it's useful to still support the mode where data is not persisted?

Reply:
I think it won't harm to persist the data, however, do we really need it? Do we have a test that restarts the driver itself so it looses data? I must miss something here.

Reply:
I read the code first and then your comments about testing :-). IMO, the driver should create `/csi-data-dir` if it does not exist.

Reply:
Fixing in https://github.com/kubernetes-csi/csi-driver-host-path/pull/73.

For e2e testing, the only potential issue is that the data will leak on the node afterwards?

Reply:
Updated to latest tag for hostpath driver to switch back to non-persistent data on driver restarts.